yaptide/routes/simulation_routes.py

```diff
@@ -54,6 +55,38 @@ def post(user: UserModel):
         )
 
 
+class ConvertInputFiles(Resource):
+    """Class responsible for returning input files converted from front JSON"""
+
+    class _Schema(Schema):
+        """Class specifies API parameters"""
+
+        sim_type = fields.String(missing="shieldhit")
+
+    @staticmethod
+    @requires_auth(is_refresh=False)
+    def post(user: UserModel):
+        """Method handling input files convertion"""
+        schema = ConvertInputFiles._Schema()
+        errors: dict[str, list[str]] = schema.validate(request.args)
+        if errors:
+            return yaptide_response(message="Wrong parameters", code=400, content=errors)
+        param_dict: dict = schema.load(request.args)
+
+        json_data: dict = request.get_json(force=True)
+        if not json_data:
+            return yaptide_response(message="No JSON in body", code=400)
+
+        task = convert_input_files.delay(param_dict=param_dict, raw_input_dict=json_data)
+        result: dict = task.wait(timeout=None, interval=0.5)
```

Comment:
```suggestion
        result: dict = task.wait()
```

no need to specify defaults

Reply:
changed manually in all occurances